src/main/resources/config/application.yml

```diff
@@ -124,6 +124,12 @@ rest:
     wait-time-millis: "\${WORKFLOW_HUB_WAIT_TIME_MILLIS:500}"
     print-body-when-error: "\${WORKFLOW_HUB_PRINT_BODY_WHEN_ERROR:true}"
 
+  pu-sil:
+    base-url: "\${PU_SIL_BASE_URL:http://localhost:8080/pusil}"
```

Comment:
you should create a PR on aks-deploy to set this variable on workflow-worker application